topics/html/01-intro/01-your-first-website/README.md

```diff
@@ -1,119 +1,121 @@
-# Tu primer sitio web
+# Sua primeira página Web
 
-- Tipo: `lectura`
-- Formato: `self-paced`
-- Duración: `20min`
+- Tipo: `leitura`
+- Formato: `individual`
+- Duração: `20min`
 
 ## Objetivos
 
-- Conocer los principales elementos y etiquetas HTML.
-- Aprender a darle dinamismo a una web utilizando JavaScript.
-- Tener un primer acercamiento al mundo del código, creando tu primer sitio
-  web.
+- Conhecer os pricipais elementos e tags HTML
+- Aprender a dar dinamismo a uma página web usando JavaScript
+- Ter uma aproximação ao mundo do código, criando seu primeiro site web
 
 ***
 
-El texto a continuación es una traducción al español, con ciertos ajustes, del
-capítulo 5 de [JavaScript for kids](http://pepa.holla.cz/wp-content/uploads/2015/11/JavaScript-for-Kids.pdf),
-Nick Morgan, 2015; y de [Eloquent JavaScript](http://eloquentjavascript.net/),
+O texto abaixo é uma tradução para o português, com alguns ajustes, do capítulo
+5 de [JavaScript for
+kids](http://pepa.holla.cz/wp-content/uploads/2015/11/JavaScript-for-Kids.pdf),
+Nick Morgan, 2015; e de [Eloquent JavaScript](http://eloquentjavascript.net/),
```

Comment:
Change the Eloquent JavaScript link to -> http://braziljs.github.io/eloquente-javascript